src/components/ExpenseForm/ExpenseForm.jsx

```diff
@@ -1,23 +1,48 @@
 import React, { useState, useEffect } from 'react'
 import { DataService } from '../../services/DataService'
 import DatePicker from 'react-datepicker'
-import { expensesUrl } from '../../static/apiconfig'
+import ChooseCategory from './ChooseCategory/ChooseCategory'
+import ChooseSubcategory from './ChooseSubcategory/ChooseSubcategory'
+import { CATEGORIES_URL, EXPENSES_URL } from '../../static/apiconfig'
 
 import 'react-datepicker/dist/react-datepicker.css'
 
 import * as Styled from './style'
 
-const  ExpenseForm = () => {
+const ExpenseForm = () => {
   const [cost, setCost] = useState('')
-  const [category, setCategory] = useState('')
+  const [categories, setCategories] = useState()
```

Comment:
Isn't it more appropriate and safer to apply initial value as empty array here and similar states?

Reply:
Done